01-front-end-basics/templating-handlebars/27-28/readme.md

```diff
@@ -19,110 +21,91 @@
 * Separate markup from logic. Remember this?
 
   ```js
-  var trackHtml = '<p><strong>' + trackData.name + '</strong> by ' + trackData.artist + '</p>';
-  $results.append(trackHtml);
+  $('#student-list').append('<li class="student">' + firstName + ' - ' + lastName + '</li>');
   ```
 
   * When appending new HTML elements to the page, the string of elements to append will only get longer as you begin to write more complex markup.
+
   * Wouldn't it be nice if the HTML structure was already set up for us? That's where templating comes in!
 
 * Maximize code reusability and maintainability.
 
   * If you need to change your HTML structure for elements you're creating and displaying (e.g. adding an additional class name to your Spotify tracks), all you have to do is change the template!
 
-## Handlebars Setup
+## A Helpful Metaphor
 
-1. Add the Handlebars CDN to your `index.html` (remember you can go to <a href="https://cdnjs.com" target="_blank">cdnjs</a> to search for CDNs). Make sure to require Handlebars before your custom script file.
+Think of a **template** as a *mold*, the **data** passed in as *plastic* poured in, and the resulting html string that is generated as a *cast*. 
 
-  ```html
-  <head>
-    <!-- meta tags, title, css links -->
-    <!-- ... -->
+<img srd="http://ecx.images-amazon.com/images/I/81GytL7BBoL._SY355_.jpg" alt="novelty cake tin - bug shaped cakes" width="200">
 
-    <!-- jquery -->
-    <script type="text/javascript" src="http://code.jquery.com/jquery-2.2.2.min.js" integrity="sha256-36cp2Co+/62rEAAYHLmRCPIych47CvdM+uTBJwSzWjI=" crossorigin="anonymous"></script>
 
-    <!-- handlebars -->
-    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.min.js"></script>
+## Explore Handlebars
 
-    <!-- custom script -->
-    <script type="text/javascript" src="js/app.js"></script>
-  </head>
-  ```
+We'll use the [HTML strings lab](https://github.com/SF-WDI-LABS/html_strings.git) as our starting point. Check out the `handelbars` branch. 
```

Comment:
* Linking to a `.git` doesn't work.
* Please check spelling on `handlebars`  
* you might also just want to link the branch itself .. ?